src/parsers/directive.parser.ts

```diff
@@ -18,7 +18,10 @@ export class DirectiveParser extends AbstractTemplateParser implements ParserInt
 		template = this._normalizeTemplateAttributes(template);
 		$(template)
 			.find('[translate]')
-			.each((i: number, element: CheerioElement) => {
+            .contents() // get child nodes
+            .filter(function() { // only keep text nodes
+				return this.nodeType === 3; // node type 3 = text node
```

Comment:
You'd want to be able to have html in your keys if you (like me) use keys as base language. 

Extract snippet from my app:
```
{
	"Log in to your account": "",
	"Listen to your audiobooks on the go": "",
	"Browse and purchase audiobooks": "",
	"Log in": "",
	"Skip log in": "",
	"This text is bold": ""
```

I don't have any HTML in the keys, but I would like the option.

You are right about the `text()` thought. Should use `html()`.

I honestly don't understand why anyone would want to use made up keys, instead of actual strings, like seems to promote? Am I missing something?